omod-common/src/main/java/org/openmrs/module/web/extension/AdministrationSectionExt.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.web.extension;
+
+import org.openmrs.module.Extension;
+
+import java.util.Map;
+
+public abstract class AdministrationSectionExt extends Extension {
```

Comment:
Why did we copy this here? Don't we already have it in another module?

Reply:
I've described this in the PR description.
In short, if I didn't include this class, the `SwaggerSpecificationCreatorTest` test would fail with `NoClassDefFoundError`. Can you tell me how can I add ` org.openmrs.module.web` module to be seen in this test?

Reply:
@dkayiwa Should I include `openmrs-module-legacyui` module as a dependency of `rest-omod-2.0` to make this test pass?